docs/resources/dashboard.md

```diff
@@ -878,6 +879,7 @@ Nested `widget` blocks have the following structure:
       - `q`: (Required) The event query to use in the widget.
       - `tags_execution`: (Optional) The execution method for multi-value filters.
     - `yaxis`: (Optional) Nested block describing the Y-Axis Controls. The structure of this block is described [below](dashboard.html#nested-widget-axis-blocks)
+    - `right_yaxis`: (Optional) Nested block describing the right Y-Axis Controls. See the `on_right_axis` property for which request will use this axis. The structure of this block is described [below](dashboard.html#nested-widget-axis-blocks)
```

Comment:
I'm not sure I follow this:
> See the `on_right_axis` property for which request will use this axis.

Is the goal to mention that in order for this to take effect you need `on_right_yaxis` field set? Maybe something like:

> Set `on_right_yaxis` to True to use this configuration

Reply:
No the fields are on 2 different objects. You have 2 axis, some will use the left, some will use the right, depending on `on_right_yaxis`.

Reply:
Ahh ok thanks